tests/unit/DynamicSegments/RequirementsCheckerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace MailPoet\DynamicSegments;
+
+use MailPoet\DynamicSegments\Filters\EmailAction;
+use MailPoet\DynamicSegments\Filters\WooCommerceCategory;
+use MailPoet\Models\DynamicSegment;
+use MailPoet\WooCommerce\Helper;
+
+class RequirementsCheckerTest extends \MailPoetUnitTest {
+
+  /** @var Helper|\PHPUnit_Framework_MockObject_MockObject */
+  private $woocommerce_helper;
+
+  /** @var RequirementsChecker */
+  private $requirements_checker;
+
+  function _before() {
+    parent::_before();
+    if (!defined('MP_SEGMENTS_TABLE')) define('MP_SEGMENTS_TABLE', '');
```

Comment:
Why is this needed? Isn't it a forgotten hotfix/workaround for something?

Reply:
This is needed. If this line is not here and the Segments model has loaded PHP complaints this constant is undefined.